src/h5web/providers/jupyter/utils.test.ts

```diff
@@ -0,0 +1,89 @@
+import { HDF5TypeClass } from '../hdf5-models';
+import { convertDtype } from './utils';
+
+describe('JupyterProvider utilities', () => {
+  describe('convertDtype', () => {
+    it('should convert unicode string dtypes', () => {
+      expect(convertDtype('|U25')).toEqual({
+        class: HDF5TypeClass.String,
+        charSet: 'H5T_CSET_UTF8',
+        strPad: 'H5T_STR_NULLPAD',
```

Comment:
Should we just remove `strPad` from the model, since we can't get the information?

Reply:
Again, I need to change in the HsdsProvider but yes it would make sense. We can restore it if the users ask for it.